log-engine-tuulbox/build.gradle.kts

```diff
@@ -14,10 +14,13 @@ kotlin {
         publishAllLibraryVariants()
     }
     js().browser()
+
     iosX64()
     iosArm32()
     iosArm64()
     macosX64()
+    macosArm64()
+    iosSimulatorArm64()
```

Comment:
Have you tested that BLE is available on the simulator?

Reply:
I have not come remotely close to that. Our other repo still needs to be updated to produce a swift package that iOS can consume. I just wanted to get everything building first.

Reply:
Removed!

Reply:
I didn't realize that we already have `iosSimulatorArm64()` in `core`. Added via a PR that an external contributed provided.

For consistency, we should probably have it here.

Sorry, for the back-and-forth.